commit 6f0c5bcdff5eec67a8fd5688e604ea1317fd26c8
Author: Brett Rudley <brudley@broadcom.com>
Date:   Wed Sep 29 17:11:11 2010 -0700

    staging: brcm80211: Fix debug section mismatch warning
    
    wl_remove() is now called from places other than the .remove field of struct pci_driver so
    do not annotate wl_remove() with __devexit.  This removes the debug section mismatch warning
    introduced by the previous nonexistant firmware patch.
    
    Signed-off-by: Brett Rudley <brudley@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/sys/wl_mac80211.c
+++ b/drivers/staging/brcm80211/sys/wl_mac80211.c
@@ -181,7 +181,7 @@ static struct pci_device_id wl_id_table[
 };
 
 MODULE_DEVICE_TABLE(pci, wl_id_table);
-static void __devexit wl_remove(struct pci_dev *pdev);
+static void wl_remove(struct pci_dev *pdev);
 #endif				/* !BCMSDIO */
 
 #ifdef BCMSDIO
@@ -1441,7 +1441,7 @@ static int wl_resume(struct pci_dev *pde
 }
 #endif				/* LINUXSTA_PS */
 
-static void __devexit wl_remove(struct pci_dev *pdev)
+static void wl_remove(struct pci_dev *pdev)
 {
 	wl_info_t *wl;
 	struct ieee80211_hw *hw;
